docs/scipp_widgets_intro.ipynb

```diff
@@ -75,9 +75,24 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 10,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
It seems you have output in your jupyter cells. Should probably clear the outputs before merging with master